OIIInspector/_OIIInspector_calls.py

```diff
@@ -0,0 +1,31 @@
+from utils import setup_arg_parser
+import sys
+
+
+
+GET_INDEX_IMAGE_PACKAGES_LIST_ARGS= {           \
+"--adress" :{                                   \
+    "help": "quay adrres of the index image",   \
+    "required":True,                            \
+    "type": str,                                \
+    }                                           \
+}
+
+
```

Comment:
Linter would probably complain of too many blank lines, but we can leave it as-is until a working CI has been set up